doc/source/user/plugins.rst

```diff
@@ -335,6 +335,57 @@ As with other plugins, :class:`~watts.PluginABCE` is easily used by::
+    Base	Nuclear	Real	2020 $/MMBTU	{{fuel_2022}}	{{fuel_2023}}   {{fuel_2024}}	{{fuel_2025}}	{{fuel_2026}}	{{fuel_2027}}	{{fuel_2028}}	{{fuel_2029}}	{{fuel_2030}}	{{fuel_2031}}	{{fuel_2032}}	{{fuel_2033}}	{{fuel_2034}}	{{fuel_2035}}	{{fuel_2036}}	{{fuel_2037}}	{{fuel_2038}}	{{fuel_2039}}	{{fuel_2040}}	{{fuel_2041}}	{{fuel_2042}}	{{fuel_2043}}	{{fuel_2044}}	{{fuel_2045}}	{{fuel_2046}}	{{fuel_2047}}	{{fuel_2048}}	{{fuel_2049}}	{{fuel_2050}}	{{fuel_2051}}	{{fuel_2052}}	{{fuel_2053}}	{{fuel_2054}}	{{fuel_2055}}	{{fuel_2056}}	{{fuel_2057}}	{{fuel_2058}}	{{fuel_2059}}	{{fuel_2060}}
+    Base	Biomass	Real	2020 $/MMBTU	5.00	5.11	5.22	5.34	5.45	5.57	5.70	5.82	5.95	6.08	6.22	6.35	6.49	6.63	6.78	6.93	7.08	7.24	7.40	7.56	7.73	7.90	8.07	8.25	8.43	8.61	8.80	9.00	9.20	9.72	10.27	10.86	11.48	12.13	12.82	13.56	14.33	15.15	16.01
+    Base	OGS	Real	2020 $/MMBTU	3.67	3.34	3.03	2.87	2.84	2.94	3.10	3.21	3.30	3.38	3.42	3.48	3.48	3.47	3.48	3.50	3.52	3.52	3.55	3.56	3.54	3.54	3.48	3.45	3.44	3.43	3.46	3.44	3.43	3.62	3.83	4.05	4.28	4.52	4.78	5.05	5.34	5.65	5.97
+    Low	Coal	Real	2020 $/MMBTU	1.41	1.38	1.42	1.35	1.31	1.29	1.28	1.29	1.29	1.29	1.29	1.29	1.30	1.30	1.32	1.33	1.34	1.34	1.36	1.37	1.37	1.38	1.39	1.40	1.41	1.42	1.41	1.42	1.43	1.51	1.59	1.68	1.78	1.88	1.99	2.10	2.22	2.35	2.48
+    Low	NG	Real	2020 $/MMBTU	3.30	3.00	2.73	2.58	2.56	2.64	2.79	2.89	2.97	3.05	3.07	3.13	3.13	3.13	3.14	3.15	3.17	3.17	3.20	3.20	3.18	3.19	3.13	3.11	3.10	3.09	3.11	3.09	3.09	3.26	3.45	3.64	3.85	4.07	4.30	4.55	4.81	5.08	5.37
+    Low	Petroleum	Real	2020 $/MMBTU	10.46	9.03	9.79	9.98	10.22	10.48	10.71	10.81	11.02	11.24	11.41	11.53	11.64	11.75	11.91	12.06	12.21	12.25	12.51	12.63	12.68	12.89	13.14	13.21	13.39	13.45	13.43	13.45	13.40	14.16	14.97	15.82	16.73	17.68	18.69	19.75	20.88	22.07	23.33
+    Low	Nuclear	Real	2020 $/MMBTU	0.62	0.62	0.62	0.62	0.62	0.62	0.62	0.63	0.63	0.63	0.63	0.63	0.63	0.41	0.41	0.41	0.41	0.41	0.41	0.41	0.41	0.41	0.41	0.41	0.41	0.41	0.41	0.41	0.41	0.41	0.41	0.41	0.41	0.41	0.41	0.41	1.03	1.09	1.15
+    Low	Biomass	Real	2020 $/MMBTU	4.50	4.60	4.70	4.80	4.91	5.02	5.13	5.24	5.36	5.47	5.59	5.72	5.84	5.97	6.10	6.24	6.37	6.51	6.66	6.80	6.95	7.11	7.26	7.42	7.59	7.75	7.92	8.10	8.28	8.75	9.25	9.77	10.33	10.92	11.54	12.20	12.90	13.63	14.41
+    Low	OGS	Real	2020 $/MMBTU	3.30	3.00	2.73	2.58	2.56	2.64	2.79	2.89	2.97	3.05	3.07	3.13	3.13	3.13	3.14	3.15	3.17	3.17	3.20	3.20	3.18	3.19	3.13	3.11	3.10	3.09	3.11	3.09	3.09	3.26	3.45	3.64	3.85	4.07	4.30	4.55	4.81	5.08	5.37
+        
+Before running the A-LEAF plugin, you need to specify the directory that the
+executable and the license key are in (they must be in the same directory). This
+can be done by adding the ``A-LEAF_DIR`` variable to the environment or by
+explicitly specifying the path in the Python script as::
+
+    aleaf_plugin = watts.PluginALEAF(
+        'aleaf_template',
+        executable="/path/to/A-LEAF"
```

Comment:
I expect A-LEAF will need also to additional input files ( extra_inputs=[]) and potentially extra templates.